helpers/deploymentConfig.ts

```diff
@@ -3093,18 +3071,38 @@ export const globalConfig: NetworkConfig = {
             reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
             vTokenReceiver: preconfiguredAddresses.sepolia.VTreasury,
           },
+          {
+            name: "Venus sfrxETH (Liquid Staked ETH)",
+            asset: "sfrxETH",
+            symbol: "vsfrxETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.09", 18),
+            jumpMultiplierPerYear: convertToUnit("3", 18),
+            kink_: convertToUnit("0.40", 18),
+            collateralFactor: convertToUnit("0.9", 18),
+            liquidationThreshold: convertToUnit("0.93", 18),
+            reserveFactor: convertToUnit("0.2", 18),
+            initialSupply: convertToUnit("1", 18),
+            supplyCap: convertToUnit(10_000, 18),
+            borrowCap: convertToUnit(1_000, 18),
+            reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
+            vTokenReceiver: preconfiguredAddresses.sepolia.VTreasury,
+          },
         ],
         rewards: [
           // XVS Rewards Over 90 days (648000 blocks)
           // WSTETH:  144 XVS for Suppliers
           //          0 XVS for Borrowers
           // WETH:    165 XVS for Suppliers
           //          385 XVS for Borrowers
+          // sfrxETH: 2400 XVS for Suppliers
+          //          0 XVS for Borrowers
           {
             asset: "XVS",
-            markets: ["wstETH", "WETH"],
-            supplySpeeds: ["222222222222222", "254629629629629"],
-            borrowSpeeds: ["0", "594135802469135"],
+            markets: ["wstETH", "WETH", "sfrxETH"],
+            supplySpeeds: ["222222222222222", "254629629629629", "3703703703703703"],
```

Comment:
Here also if we are distributing 24 XVS tokens then we should also decrease the speed i.e. `3703703703703703/100`